portfolio/src/main/webapp/index.html

```diff
@@ -5,7 +5,7 @@
     <meta charset="UTF-8">
     <title>Santiago Bukowitz</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <script src="script_2.js"></script>
```

Comment:
you can name this with what it does so that it will not be confusing to you in the future